src/app/app.component.ts

```diff
@@ -28,30 +30,36 @@ export class AppComponent {
 
   @ViewChild(DialogComponent, {static: false})
   settingsDialog: DialogComponent | undefined;
+
   settingsSubject$: BehaviorSubject<boolean> = new BehaviorSubject(false);
 
+  ngOnInit(): void {
+    this.isDarkThemeEnabled = this.themeService.getActiveTheme === Theme.Dark;
```

Comment:
I think the implementation of the `getActiveTheme` function should return the `getPreferredBrowserTheme` if there's no theme preference stored in local storage instead of the fallback default (light). Thoughts?

Reply:
I agree with that.